gabi_test.go

```diff
@@ -235,7 +236,8 @@ func TestProofU(t *testing.T) {
 	secret, _ := common.RandomBigInt(DefaultSystemParameters[keylength].Lm)
 
 	b := NewCredentialBuilder(testPubK, context, secret, nonce2, nil)
-	proofU := b.CreateProof(createChallenge(context, nonce1, b.Commit(map[string]*big.Int{"secretkey": secret}), false))
+	contributions, _ := b.Commit(map[string]*big.Int{"secretkey": secret}) // returns error only for interface, never errors
```

Comment:
In testing code I prefer that we always check for errors. Even though it does not error now, it could error in the future, e.g. when we add checking for errors during random number generation.

Reply:
fixed